resources/words/add_tag.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
```

Comment:
Could you please explain what exactly this script does and how to use it? Which kind of input does it expect and what's the actual gain? I thought it auto-translates, but I see no translation code :man_shrugging:

Reply:
I think the italian word file is a 1-to-1 translation of the english file. So the first line of the italian translation matches the first word of the english file etc. This script adds - as far as I understand it - the difficulty level (#h etc.) at the end.

Reply:
But there's no actual translation happening in this script, right? Or am i blind?

Reply:
As far as I understand it - no there is not. It takes an already translated language file and just adds the tag at the end.

Reply:
I will make another PR for this script eventually